services/web/README.md

```diff
@@ -43,6 +43,31 @@ This feature is useful for organizations that want to provide third party or cus
 It's important to note that the feature at the moment is only capable of providing static (js, mjs, e.g.) web applications
 and does not support injection of dynamic web applications (custom dynamic backends).
 
+### Loading Themes
+
+Web themes are loaded, if added in the Infinite Scale source code, at build-time from
+`<ocis_repo>/services/web/assets/themes`.
+This cannot be manipulated at runtime.
+
+Additionally, the administrator can provide custom themes by storing it in the path defined by the environment
+variable `WEB_ASSET_THEMES_PATH`.
+
+This environment variable defaults to the Infinite Scale base data directory `$OCIS_BASE_DATA_PATH/web/assets/themes`,
+but can be redefined with any path set manually.
+
```

Comment:
```suggestion
```
We can omit this, as it will be part of the envvar description as proposed. The envvar description change applies to both `WEB_ASSET_THEMES_PATH` and `WEB_UI_THEME_PATH`.